### DISCUSSION TITLE
single-line Panel/Layout possible?

### DISCUSSION THREAD

**Discussion by foozmeat** [Textualize/rich#1099] [Q&A] [answered]
I have an existing curses application that needs to make the most of limited screen size. If I were to convert it to rich I'd need to be able to make single-line panels but I can't figure out how to collapse the area reserved for the border. Setting a Layout size < 3 results in the panel content disappearing. Is it possible? Here's an example where the `info_panel` and `status_panel` disappear.

```python
from rich import print
from rich.layout import Layout
from rich.panel import Panel

layout = Layout(name="root")
info_panel = Panel("single-line info")

layout.split(
        Layout(info_panel, name="info", size=1),
        Layout(name="main")
)

layout["main"].split(
        Layout(name="lists"),
        Layout(name="io", ratio=2),
        direction="horizontal"
)

status_panel = Panel("single-line status")
layout["io"].split(
        Layout(status_panel, name="status", size=1),
        Layout(name="log"),
)

layout["lists"].split(
        Layout(name="devices"),
        Layout(name="tasks"),
)

print(layout)
```

**Comment by willmcgugan** [comment, marked as answer]
Panels have a border around them. If you just want text, the use a plain string, or a `rich.text.Text` instance.

**Reply by foozmeat** [reply]
Of course! Sorry I was overthinking things. Appreciate the quick reply.
